### DISCUSSION TITLE
Blog post about creating a DSL with Langium

### DISCUSSION THREAD

**Discussion by ymc9** [eclipse-langium/langium#1353] [Show and tell]
We love Langium, and it helped us tremendously in building our DSL. I've created a blog post introducing the general process of creating a language using Langium. I hope this can help people who might want to embark on the journey have an easier start.

https://dev.to/zenstack/how-much-work-does-it-take-to-build-a-programming-language-37ji

**Comment by spoenemann** [comment]
Thank you very much @ymc9 for the great article, and also for your [sponsorship](https://github.com/sponsors/TypeFox)! ❤️
